src/ssr-template-helper.js

```diff
@@ -105,10 +105,13 @@ export class SsrTemplateHelper {
         typeof data['html'] === 'string',
         'Server side html response must be defined'
       );
-      renderTemplatePromise = this.templates_.findAndSetHtmlForTemplate(
-        element,
-        /** @type {string} */ (data['html'])
-      );
+      renderTemplatePromise = this.viewer_.isTrustedViewer().then(trusted => {
+        userAssert(trusted, 'May only ssr from trusted viewers');
```

Comment:
Minor: Let's include `element` in the assert to make this error message more meaningful.

```js
userAssert(trusted, 'Refused to apply SSR in untrusted viewer: ', element);
```

Reply:
Done